packages/appeals-service-api/src/db/schema.prisma

```diff
@@ -10,8 +7,120 @@ datasource db {
+  serviceUserId Int? // unique, with nulls allowed, is enforced in SQL, see first migration script & https://stackoverflow.com/a/767702
+
+  // LPA user fields
+  // the LPA this user belongs to
+  lpaCode    String?
+  // whether this LPA user is an admin for that LPA
+  isLpaAdmin Boolean?
+  // the status of this user, e.g. have they logged in and confirmed their email?
+  status     String? // added | confirmed | removed
+
+  // appeals this user is linked to, by being the appellant or agent, for example
+  Appeals AppealToUser[]
+
+  SecurityToken SecurityToken?
+}
+
+// SecurityToken is used as a mechanism for users to login
+// They enter their email, a token is generated, emailed to the user, and they enter it to login
+// A user only ever has one token, which gets updated when a new token is required
+model SecurityToken {
+  id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+
+  // link to a user
+  appealUserId String     @unique @db.UniqueIdentifier
+  AppealUser   AppealUser @relation(fields: [appealUserId], references: [id])
+
+  // the generated token the user can login with
+  token            String
+  // the time the token was generated, used for checking token expiry
+  tokenGeneratedAt DateTime
+  // the number of attempts at entering the code since this token was last generated
+  // this MUST be updated in a transaction to ensure atomic operation
+  // see https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#atomic-number-operations
+  attempts         Int      @default(0)
+  // the action the code was generated for, e.g. confirmEmail
+  action           String?
+}
+
+// Appeal is the base appeal model, used to link references to other entities, which includes, for example:
+// - an appeal submission authored in the front office
+// - a case published by the back office
+// - an LPA questionnaire submission
+model Appeal {
+  id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+
+  // users linked to this appeal, such as the appellant or agent
+  Users AppealToUser[]
+
+  // reference to an appeal submission in the Cosmos DB `appeals` collection
+  // we sync across two properties, as below
+  // this is so we can query for draft submissions where the deadline date has passed (to delete them)
+  legacyAppealSubmissionId           String?
+  // the deadline date of the appeal submission in the Cosmos DB `appeals` collection
+  legacyAppealSubmissionDecisionDate DateTime?
+  // the state of the appeal submission in the Cosmos DB `appeals` collection
+  legacyAppealSubmissionState        String? // DRAFT || SUBMITTED
+
+  // link to case (from Back Office)
+  AppealCase AppealCase?
+
+  // todo: in the future link to appeal submission and other data
+}
+
+// AppealToUser represents the relationships between AppealUser and Appeal
+// This is a many-to-many relationship.
+model AppealToUser {
+  // relation to an Appeal
+  appealId   String     @unique @db.UniqueIdentifier
```

Comment:
I don't think `appealId` would be unique in this instance as we can have multiple users (e.g. agent & appellant) linked to the same appeal.

Reply:
Definitely a copy-paste error!